herbert_core/utilities/classes/@IX_dataset_2d/private/plot_2d_nd_oc_.m

```diff
@@ -0,0 +1,55 @@
+function [fig_,axes_,plot_] = plot_2d_nd_oc_(w,nout,type,opt,varargin)
```

Comment:
Additional function name component oc_ is not documented - comments here look cut and paste from the one without oc.

Reply:
added description